.github/workflows/ci-cd.yaml

```
@@ -178,6 +178,25 @@ jobs:
           cache-from: type=gha
           cache-to: type=gha,mode=max
       
+      # Upload debug symbols to Sentry
+      - name: Setup Sentry CLI
+        if: secrets.SENTRY_AUTH_TOKEN != ''
```

actionlint (1.7.9)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)